tests/tasks/generators/manual-deployment/deploy-benchmark-payload.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: deploy-benchmark-payload
```

Comment:
Although we should be relying on clusterLoader2 for these tasks, but we can keep it for now and switch to clusterLoader later. 
However, we should make the task name more descriptive and clear as these tasks should clearly reflect what they are doing.

Reply:
Agree. Done.